tests/xharness/Logging/ILogs.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace xharness.Logging {
+	public interface ILogs : IList<Log>, IDisposable {
+		string Directory { get; set; }
+
+		// Create a new log backed with a file
+		ILogFile Create (string filename, string name, bool? timestamp = null);
+
+		// Adds an existing file to this collection of logs.
+		// If the file is not inside the log directory, then it's copied there.
+		// 'path' must be a full path to the file.
+		ILogFile AddFile (string path);
+
+		// Adds an existing file to this collection of logs.
+		// If the file is not inside the log directory, then it's copied there.
+		// 'path' must be a full path to the file.
```

Comment:
Not sure what is the Xamarin custom on documenting methods, but maybe it would be better to do XML docs?

Reply:
Moving code, wrote by rolf